Rocket.ChatTests/Managers/DatabaseManagerSpec.swift

```diff
@@ -9,26 +9,40 @@
 import XCTest
 @testable import Rocket_Chat
 
+private let testServers = [[
```

Comment:
Is this global 😱 ? Should we move it to the DatabaseManager extension?

Reply:
You actually can't add properties to the extensions... let's leave it there until we find a better solution?

Reply:
Always forget that 👍